### PR TITLE
Added support for  added_cond_kwargs 

### PR DESCRIPTION
**Pull request by tthakkal**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

**Review by skavulya — APPROVED** [review]
Looks good to me. Thanks @tthakkal

File: optimum/habana/trl/models/modeling_sd_base.py
Comment (skavulya) [inline optimum/habana/trl/models/modeling_sd_base.py:1]: @tthakkal Please add support for added_cond_kwargs in unet_hpu() call

https://github.com/skavulya/optimum-habana/blob/ee2f8d5fa343bdb7c7b19efc5e71aa9a782fa170/optimum/habana/trl/models/modeling_sd_base.py#L370

See: https://github.com/huggingface/optimum-habana/blob/6f889a85792ba85a8d24b80497220ae0e7b91c54/optimum/habana/diffusers/pipelines/stable_diffusion/pipeline_stable_diffusion.py#L449

Comment (tthakkal) [reply optimum/habana/trl/models/modeling_sd_base.py:1]: Updated, but we need to sync your fork `huggingface/optimum-habana` first

**Review by skavulya — APPROVED** [review]

File: optimum/habana/trl/models/modeling_sd_base.py
Comment (skavulya) [reply optimum/habana/trl/models/modeling_sd_base.py:1]: Let me sync my fork

**Review by skavulya — APPROVED** [review]

**Pull request merged**